### DISCUSSION TITLE
Filen

### DISCUSSION THREAD

**Discussion by plainasday** [elrumo/macOS_Big_Sur_icons_replacements#1356] [Icon request] [answered]
Icon request for Filen cloud storage. 

Website: https://filen.io
iOS app: https://apps.apple.com/us/app/filen-cloud-storage/id1549224518
Press kit: https://cdn.filen.io/PressKit.zip

Thanks!

**Comment by bunkerdesign** [comment, marked as answer]
<img width="512" alt="Filen cloud storage" src="https://user-images.githubusercontent.com/71460900/153431386-5223d56c-9365-43d8-a2bc-f9f665e0b0ef.png">
Here

**Reply by plainasday** [reply]
Thank you very much!

**Reply by bunkerdesign** [reply]
You are welcome! Could you please mark the request as answered?
